plugins/circleci/src/components/BuildsPage/lib/CITable/CITable.tsx

```diff
@@ -169,7 +169,7 @@ const generatedColumns: TableColumn[] = [
     field: 'buildName',
     highlight: true,
     render: (row: Partial<CITableBuildInfo>) => (
-      <Link href={row?.buildUrl} target="_blank">
+      <Link to={row?.buildUrl!} target="_blank">
```

Comment:
exclamation intentional?

Reply:
Yeah. This ensures the row.buildUrl exists.

Reply:
@adamdmharvey must I remove this exclamation point yer?